test/inductor/test_compiled_optimizers.py

```diff
@@ -80,10 +162,29 @@ def test_fn(self):
             opt_eager.step()
             opt_eager.step()
 
+        # Note on tolerances:
+        # test_adadelta_foreach_cuda_rho_weight_decay
```

Comment:
if the tolerance is only for one config, could we special case and change the tolerance only in that case?

Reply:
One way is to pass atol and rtol into make_test